README.rst

```diff
@@ -13,7 +13,7 @@ Refer to their respective documentation for more information.
    :maxdepth: 2
    :caption: Contents:
 
-   bsdlib/README
+   nrf_modem/README
```

Comment:
These entries are alphabetic, so need to move down.

Reply:
Fixed